src/js/contexts/AnnounceContext/announce-context.stories.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { storiesOf } from '@storybook/react';
+import { grommet } from 'grommet/themes';
+import { Box, Grommet, Text, AnnounceContext } from 'grommet';
+
+class Announcer extends Component {
+  static propTypes = {
+    announce: PropTypes.func.isRequired,
```

Comment:
Any reason not to include message, role, and politenessSetting?

Reply:
Addressed in recent commit. Thanks !!